js/src/layers/Layer.js

```diff
@@ -60,6 +91,12 @@ export class LeafletLayerView extends utils.LeafletWidgetView {
         this.bind_popup(value);
       });
       this.update_pane();
+      this;
+      this.subitem_views = new widgets.ViewList(
+        this.add_subitem_model,
+        this.remove_subitem_view,
+        this
+      );
```

Comment:
You need to call `update` on the view list at least once at creation, so that you populate the sub-items properly.

Reply:
Ok thanks. this should have been fix in commit [b7e97f9](https://github.com/jupyter-widgets/ipyleaflet/pull/1011/commits/b7e97f9fac9aa201f29b6be649f1947106db5af1)